ajax/libs/json2csv.cjs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "json2csv.cjs",
+  "filename": "json2csv.cjs.js",
```

Comment:
UMD file is the one for a browser, please use `json2csv.umd.min.js`, we will minify it :)